scripts/player.gd

```diff
@@ -1,33 +1,53 @@
 extends CharacterBody2D
 
 
+#Exports variable
+@export var yes = 1
+#roations vars
+@export var speed = 400
+@export var rotation_speed = 1.5
 
-const SPEED = 300.0
-const JUMP_VELOCITY = -400.0
 
-# Get the gravity from the project settings to be synced with RigidBody nodes.
-var gravity = ProjectSettings.get_setting("physics/2d/default_gravity")
 
+#Variables for controlling the movement
+const max_speed = 50
+const accel = 50
+const friction = 20
 
+#input variable
+var input = Vector2.ZERO
+var rotation_direction = 0
+
+#runs when the game starts and runs our player_move func.
 func _physics_process(delta):
-	# Add the gravity.
-	print("Player Script Active");
-	if not is_on_floor():
-		velocity.y += gravity * delta
-		print("Falling");
-
-	# Handle jump.
-	if Input.is_action_just_pressed("ui_accept") and is_on_floor():
-		velocity.y = JUMP_VELOCITY
-
-	# Get the input direction and handle the movement/deceleration.
-	# As good practice, you should replace UI actions with custom gameplay actions.
-	var direction = Input.get_axis("ui_left", "ui_right")
-	if direction:
-		velocity.x = direction * SPEED
-		print("Direction is: " + direction);
-		print($"Speed is: {SPEED}");
+	player_movement(delta)
+
+func get_input():
+	rotation_direction = Input.get_axis("ui_left", "ui_right")
```

Comment:
where are ui_left and ui_right defined?

Reply:
They don't need to be defined until we need to change the movement keys. The engine knows that ui_direction means to use that arrow key.